### PR TITLE
Create introduction/introduction.md

### PR DESCRIPTION
**Pull request by rrrutledge**
This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).

### CODE REVIEW

File: introduction/introduction.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
```
Comment (gruetter) [inline introduction/introduction.md]: s/inner source/InnerSource/g

File: introduction/introduction.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Inner source is the strategy of applying open source practices and principles to internal software development.
```
Comment (gruetter) [inline introduction/introduction.md]: > Inner source is the strategy of ...

We have always coined InnerSource as being _the application of_ open source practices and principles. I propose we stick with that.

File: introduction/introduction.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+This strategy results in producing the highest quality software that is responsive and nimble to the changing needs of its many internal consumers.
```
Comment (gruetter) [inline introduction/introduction.md]: While I do agree that InnerSource has an impact on quality, that wouldn't be the first benefit that comes to mind. How about we mention the following instead:

- reduce barriers for collaboration / make collaboration across silos efficient
- efficiently development of shared assets
- efficiently share knowledge
- increase innovation

File: introduction/introduction.md (hunk 1 above)
Comment (rrrutledge) [reply introduction/introduction.md]: Thanks for the suggestion.  The proposed casing is not my favorite.  I noticed it use in the Trusted Committer articles but didn't bother bringing it up.

Some background ...

... I think that we like to use inner source as both a noun and a verb.  I think that we want to spell it the same way both times.  InnerSource works great as a noun, but I really run into trouble when using it as a verb.  I don't feel good about mixed casing on verb tenses like InnerSourcing and InnerSourced, so that leaves me with calling it "inner source" everywhere.

There was a small discussion of the casing to use in [this survey](https://docs.google.com/forms/d/1Z-oe1WdER-_wzW4b0Yo5RxcRLTpf8BnJk-D_k14m65I/edit#responses).

File: introduction/introduction.md (hunk 3 above)
Comment (rrrutledge) [reply introduction/introduction.md]: Kept it to one sentence, but tried to include (or hint at) these ideas:

> reduce barriers for collaboration

"wide"

> efficiently development of shared assets

"effective"

> increase innovation
 
"responsive and nimble"

File: introduction/introduction.md (hunk 2 above)
Comment (rrrutledge) [reply introduction/introduction.md]: Done.

File: introduction/introduction.md (hunk 3 above)
Comment (KathrynKLewis) [reply introduction/introduction.md]: Suggest rewording first sentence so reader understands that inner source supports open source practices at the enterprise or organization level. In my opinion, the term "internal software development" is unclear. 
REVISION: 
Inner source is the strategy of applying open source practices and principles to internal software development, i.e. within an enterprise or organization. The enterprise or organization still develops proprietary software, but internally opens up its development to allow for accessibility and reuse.

Comment (rrrutledge) [reply introduction/introduction.md]: I like the thrust of this revision.  I reworded both of your sentences to incorporate the idea of what I understood you saying but with the following constraints in sentence structure.

- **1st sentence**: No extra clauses.
- **2nd sentence**: Have the active noun in the sentence be the software instead of the company.

File: introduction/introduction.md (hunk 1 above)
Comment (demianbrecht) [reply introduction/introduction.md]: I know that I'm far behind the curve here and my opinion doesn't carry much (if any) weight but FWIW I'm definitely a solid -1 to InnerSource. The major reason for me is mentioned in the [doc](https://docs.google.com/document/d/1nmP4-Gj4QkkRkMFaZNrnt7KmlKVJOu0zI-wxTa-6RGo/edit) linked in the survey: parity with "open source". I also agree with @rrrutledge that when used as a verb, it's really off putting. I'd also imagine that folks evangelizing inner sourcing in any capacity would use "inner sourcing" (I know I have) and we'd therefore breed disparity right out of the gate.

IMHO, unless there's a legal reason that InnerSource needs to be used, a move to "inner source" should be pursued.

My 2c anyways :)

File: introduction/introduction.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the application of open source practices and principles to software development within the enterprise.
```
Comment (demianbrecht) [inline introduction/introduction.md]: Maybe a slight rewording? Something like:

> Inner source is the application of open source practices and principles to proprietary or closed source software development.

As is, it reads to me like the target audience is for-profit companies (enterprises). There's no reason why inner sourcing practices can't be employed elsewhere too (non-profits, etc).

File: introduction/introduction.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+Inner source software remains proprietary to the company, but within it is open for anyone to use and contribute.
```
Comment (demianbrecht) [inline introduction/introduction.md]: "contribute to"?

File: introduction/introduction.md (hunk 4 above)
Comment (KathrynKLewis) [reply introduction/introduction.md]: I agree with the comment Demian is making about nonprofits etc but I don't think the word "enterprise" is off putting or confusing, but I do think the term "closed source software development" is confusing. As a reader of the original phrasing, I can infer the meaning of inner source and expand that to see its relevance in any organization.

Comment (demianbrecht) [reply introduction/introduction.md]: Yeah that's fair. Perhaps I was being a tad too pedantic there :)

File: introduction/introduction.md (hunk 1 above)
Comment (rrrutledge) [reply introduction/introduction.md]: I'll leave this conversation open a bit longer in case there's any more comment but at this point am thinking of resolving it without change to the text.

File: introduction/introduction.md (hunk 5 above)
Comment (rrrutledge) [reply introduction/introduction.md]: Adjusted to `contribute to it`.

File: introduction/introduction.md (hunk 3 above)
Comment (rrrutledge) [reply introduction/introduction.md]: @gruetter @KathrynKLewis see what you think of the (proposed) incorporation of your ideas.

File: introduction/introduction.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
```
[the remark this answers is not in the document]
Comment (rrrutledge) [reply introduction/introduction.md]: You're just seeing the preview/teaser.  Log in with an O'Reilly Safari account to see the rest!  We're working on getting the videos released to all members of the InnerSource Commons but for now you need that O'Reilly Safari account to see them.

File: introduction/introduction.md (hunk 1 above)
Comment (gruetter) [reply introduction/introduction.md]: Just for the record, I am also in the _inner source_ camp! I just used _InnerSource_ because we have been using is more or less consistently everywhere else, AFAIK.

File: introduction/introduction.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the application of open source practices and principles to software development within the enterprise.
+Inner source software remains proprietary to the company, but within it is open for anyone to use it and contribute to it.
+This strategy enables wide and effective collaboration, producing software that is responsive and nimble to the changing needs of its many internal stakeholders.
```
Comment (gruetter) [inline introduction/introduction.md]: Hmm ... I think InnerSource/inner source is not primarily about the properties of the software that it produces (nimbleness or responsiveness) but rather about the process of creating it. I propose to remove that and add more info on the quality of the process and the overall outcome for the organisation (more efficiency, better collaboration) and the involved individuals (learning) instead.

File: introduction/introduction.md (hunk 1 above)
Comment (rrrutledge) [reply introduction/introduction.md]: Oh, nice!  What about going back through the Trusted Committer text and changing to _inner source_ instead?

File: introduction/introduction.md (hunk 7 above)
Comment (rrrutledge) [reply introduction/introduction.md]: We've iterated on this paragraph a number of times in this PR, but I feel like my effectiveness in incorporating your great thoughts is waning.  If this paragraph needs further attention, can you open a PR with suggested text that you'd like to see?  Thanks

**Comment by lenucksi** [comment]
Just matched video and text, while they are not identical, the text catches the important points and is clear and concise.
I'd merge it, however I don't have the rights to do so and can't add myself as a reviewer to approve/disapprove.
Thus Have a Gerrit style '+2' from me. ;)

**Comment by rrrutledge** [comment]
Good enough, I will merge!  Thanks, @lenucksi.
